src/sflow_plugin/sflow_data.h

```diff
@@ -151,7 +151,8 @@ typedef struct _SFSample {
     uint8_t eth_dst[8];
 
     /* vlan */
-    uint32_t in_vlan;
+    uint32_t in_svlan;
+    uint32_t in_cvlan;
```

Comment:
Same comment as before. You can add new variable but please keep existing one.

Reply:
Fixed